css/style.css

```diff
@@ -1 +1,1091 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgb(13, 13, 17);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", sans-serif;
+}
+
+/* Uneversal css */
+
+* {
+  padding: 0;
+  margin: 0;
+  background-color: rgba(4, 11, 17, 1);
+  font-family: "Bakbak One", sans-serif;
```

Comment:
Provide fallback fonts and family types at the end of the font list. 
For example: `font-family: "Roboto", Arial, sans-serif;`

Reply:
#fixed